admin/settingstree.php

```diff
@@ -0,0 +1,45 @@
+<?php
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+/**
+ * All DokuWiki plugins to extend the admin function
+ * need to inherit from this class
+ */
+class admin_plugin_dw2pdf_settingstree extends DokuWiki_Admin_Plugin {
+	private $helper = null;
```

Comment:
I originally intended to allow plugin authors to include settingstree in their own plugin as "library". They create admin if they need, they create multiple admin for different settings (e.g. dw2pdf config in different admin then extended template config), or use it transparently to the user without admin.

Creating a not-alternative-only-just-extends conf manager would be probably confusing in many ways.
Maybe integrating settingstree to conf manager itself would be better approach for what you suggesting, but that would go with pretty major changes.